src/sentry/dynamic_sampling/__init__.py

```diff
@@ -1,10 +1,17 @@
 from .rules.base import generate_rules
-from .rules.biases.boost_environments_bias import ENVIRONMENT_GLOBS, BoostEnvironmentsRulesGenerator
-from .rules.biases.boost_key_transactions_bias import BoostKeyTransactionsRulesGenerator
-from .rules.biases.boost_latest_releases_bias import BoostLatestReleasesRulesGenerator
+from .rules.biases.boost_environments_bias import ENVIRONMENT_GLOBS
+from .rules.biases.boost_environments_bias import (
+    BoostEnvironmentsRulesGeneratorV2 as BoostEnvironmentsRulesGenerator,
+)
+from .rules.biases.boost_key_transactions_bias import (
+    BoostKeyTransactionsRulesGeneratorV2 as BoostKeyTransactionsRulesGenerator,
+)
+from .rules.biases.boost_latest_releases_bias import (
+    BoostLatestReleasesRulesGeneratorV2 as BoostLatestReleasesRulesGenerator,
+)
+from .rules.biases.ignore_health_checks_bias import HEALTH_CHECK_GLOBS
 from .rules.biases.ignore_health_checks_bias import (
-    HEALTH_CHECK_GLOBS,
-    IgnoreHealthChecksRulesGenerator,
+    IgnoreHealthChecksRulesGeneratorV2 as IgnoreHealthChecksRulesGenerator,
```

Comment:
Why not rename those classes themselves? There is no more V1, so we might as well drop the `V2` suffix entirely.

Reply:
Will do

Reply:
☝️